src/zks_provider/mod.rs

```diff
@@ -1772,11 +1772,18 @@ mod tests {
             .deploy(
                 "src/compile/test_contracts/storage/src/ValueStorage.sol",
                 "ValueStorage",
-                Some(U256::zero()),
+                Some(vec!["0".to_owned()]),
```

Comment:
we don't have to address this here, but if we're passing a vec, then I don't see the need for an Option, since no args would be the same as an empty vec?

Reply:
I agree with this, I'll file an issue if it's ok.

Reply:
Filed in #74